internal/webhooks/cluster_test.go

```diff
@@ -320,11 +361,9 @@ func TestClusterDefaultVariables(t *testing.T) {
 			// Test if defaulting works in combination with validation.
 			util.CustomDefaultValidateTest(ctx, cluster, webhook)(t)
 			// Test defaulting.
-			t.Run("default", func(t *testing.T) {
```

Comment:
nit: This was intentional as `util.CustomDefaultValidateTest` also creates sub-test-cases (named `validate-on-create`, ...)

Is there a reason why we can't keep it?